src/pages/explore.html

```diff
@@ -1,85 +1,108 @@
         </p>
     </div>
 
-    <div class="resource-info" ng-if="isTripleResource()">
-        <div class="thumb" ng-show="{{details.img}}">
-            <a href="{{details.img}}"><img ng-src="{{details.img}}" alt="details image"/></a>
+    <div class="resource-info" ng-if="resourceInfo && isTripleResource()">
+        <div class="thumb" ng-if="resourceInfo.details.img">
+            <a href="{{resourceInfo.details.img}}">
+                <img ng-src="{{resourceInfo.details.img}}" alt="details image"/>
+            </a>
         </div>
         <h1 fit-text fit-text-min="30">
-            <a href="sparql?query=describe%20{{encodeURIComponent(tripleParam)}}" gdb-tooltip="{{tripleParam}}">
-                {{getRdfStarLocalNames(tripleParam)}}
+            <a href="sparql?query=describe%20{{encodeURIComponent(resourceInfo.triple)}}"
+               gdb-tooltip="{{resourceInfo.triple}}"
+            class="triple-resource-link">
+                {{getRdfStarLocalNames(resourceInfo.triple)}}
             </a>
         </h1>
 
-        <p ng-show="{{details.comment}}" data-escape="false">{{details.comment}}</p>
+        <p ng-show="resourceInfo.details.comment" data-escape="false">{{resourceInfo.details.comment}}</p>
         <p>
-            <span>{{'target.label' | translate}}:</span>&nbsp;<a href="sparql?query=describe%20{{encodeURIComponent(tripleParam)}}">{{tripleParam}}</a>
+            <span>{{'target.label' | translate}}:</span>&nbsp;
+            <a href="sparql?query=describe%20{{encodeURIComponent(resourceInfo.triple)}}"
+               class="target-link">
+                {{resourceInfo.triple}}
+            </a>
         </p>
     </div>
 
-    <div class="pull-right">
+    <div class="pull-right" ng-if="resourceInfo">
 
         <label for="inference-select" class="mb-0">
             <select id="inference-select"
-                    ng-options="inf.id as inf.title | translate for inf in inferences" ng-model="inference"
-                    ng-change="changeInference(inference)" class="form-control form-control-sm"
-                    ng-disabled="loading || role === 'context' || getActiveRepository() === 'SYSTEM'"></select>
+                    class="form-control form-control-sm"
+                    ng-model="currentContextType"
+                    ng-change="changeInference(currentContextType)"
+                    ng-disabled="loading || role === 'context' || getActiveRepository() === 'SYSTEM'">
+                <option ng-repeat="contextType in contextTypes"
+                        value="{{contextType.id}}"
+                        ng-selected="resourceInfo.contextType.id === contextType.id">
+                    {{contextType.labelKey | translate}}
+                </option>
+            </select>
         </label>
 
-        <button type="button" class="btn btn-link same-as-btn" ng-model="sameAs"
+        <button type="button" class="btn btn-link same-as-btn"
```

Comment:
Can lower the size of the icon here because when it appears now it moves the tab content a bit lower

Reply:
fixed